### ISSUE TITLE
Republish and add to "newly instrumented" list

### ISSUE THREAD

**Issue by samreid** [phetsims/friction#74]
See #67 

> After publishing, add your instrumented simulation to the spreadsheet here: https://docs.google.com/spreadsheets/d/1pU9izdNQkd9vr8TvLAfXe_v68yh-7potH-y712FBPr8/edit#gid=0

After #72 we can republish PhET and PhET-iO brands.  On hold until #72 done.

**Comment by samreid** [comment]
@jbphet will be continuing the final steps in #70 and #71, and will oversee testing in #72.  Assigned to @jbphet, but leaving on hold until sub-issues are completed.

**Comment by jbphet** [comment]
The three issues listed above have been addressed.  I can go through the process of re-releasing this off of master, but it should have a full QA test, so I thought I'd double check that this is still needed.  Also, is there any input as to whether I should publish both phet and phet-io, or just phet-io?  It seems like doing both would be good since they'd stay in sync.

**Comment by jbphet** [comment]
Also, I want to make sure that the phet-io team agrees that this is in a state where it is ready for publication.

**Comment by samreid** [comment]
Molarity's recent RC revealed some issues that will need to be fixed before we can proceed with this sim.  See https://github.com/phetsims/QA/issues/161

**Comment by samreid** [comment]
Sounds great to me as long as our time/schedule permits.  Scheduling and timing and priority seem to be a good question for @ariel-phet @kathy-phet and/or @emily-phet.

**Comment by emily-phet** [comment]
I haven't dug too deep on this - there are a lot of issues linked...from my perspective molarity doesn't have a specific timeline. It was awaiting completion of the combobox + all the other things that happen prior to publication. Not sure exactly when we'll get to it. 

Might be good to have a general conversation about a desired flow for republishing sims with a11y and phet-io. Then when we get a little further down the road, we can see if we can get this alignment to happen for molarity, or perhaps future (other) sims.

**Comment by zepumph** [comment]
A general conversation can start with https://github.com/phetsims/phet-io/issues/1395.

**Comment by emily-phet** [comment]
This seems stale now. Processes for a11y and phet-io have evolved significantly from when this issue was started. Closing.
